### ISSUE TITLE
Setup routes for pages

### ISSUE THREAD

**Issue by jpatrici** [chingu-voyages/v39-bears-team-16#13]
Add these basic routes:
- `/` and `/home` for the homepage after login. This should be the default route
   Note: in the future, the route can be divided even further by doing `/admin/home` and `/student/home`
- `/sign-up`
- `/sign-in`
- `/forgot-password`

Optional to be added now: (can be added later on too)
- `/admin` and `/student`
    - `/class`
    - `/class/:id`
    
Blocked by #7

**Comment by jpatrici** [comment]
Add these basic routes:
-`/` and `/home` for the homepage after login. This should be the default route
- `/sign-up`
- `/sign-in`
- `/forgot-password`

Optional to be added now: (can be added later on too)
- `/admin` and `/student`
    -`/class`
    - `/class/:id`

**Comment by jpatrici** [comment]
closed because it's a duplicate of #12
